src/components/CreatePost/CreatePost.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+
+import { Kweet as KweetModel } from '../../model/kweet';
+import { Kweet } from '../Kweet';
+import {
+  KwitterButton,
+  KwitterPost,
+  KwitterTextArea,
+  ProfilePhoto
+} from './styled';
+
+export const CreatePost = (): React.ReactElement => {
+  const [post, setPost] = useState<string>('');
+  const [click, setClick] = useState<boolean>(false);
+  const [kweetList, setkweetList] = useState<KweetModel[]>([]);
+
+  const dateValue = (value: string) => new Date(value).valueOf();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const fetchUsers = await axios.get('http://localhost:3001/kweets');
```

Comment:
We should use the `kwitterApiClient` exported from `api/client` that already has set `baseUrl`. 🙂